### PR TITLE
[ISSUE #86] Add view controllers to console

### PR DESCRIPTION
**Pull request by zzxxiansheng**

### CODE REVIEW

**Comment by zzxxiansheng** [comment]
I don't know what the original ‘’topiccontroller‘’means, but the name has to be distinguished from the ‘’new topiccontroller‘’

**Comment by Pil0tXia** [comment]
@zzxxiansheng It's the same. You may reuse the old one.

**Comment by zzxxiansheng** [comment]
> @zzxxiansheng It's the same. You may reuse the old one.

Is the topiccore interface in the original topiccontroller called by the viewcontroller（my topic controller）?

**Comment by Pil0tXia** [comment]
@zzxxiansheng Your current TopicController code is just fine, no need to change it.

File: eventmesh-dashboard-console/src/main/java/org/apache/eventmesh/dashboard/console/service/registry/impl/RegistryDataServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    }
+
+    @Override
+    public Long insert(MetaEntity metaEntity) {
+        return metaMapper.insert(metaEntity);
+    }
+
+    @Override
+    public void deActive(MetaEntity metaEntity) {
+        metaMapper.deActive(metaEntity);
```
Comment (Pil0tXia) [inline eventmesh-dashboard-console/src/main/java/org/apache/eventmesh/dashboard/console/service/registry/impl/RegistryDataServiceImpl.java:51]: `deactivate` is a single word, so it should be `deactivate()` here.

File: eventmesh-dashboard-console/src/main/java/org/apache/eventmesh/dashboard/console/modle/dto/config/DynamicGetConfigDTO.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+package org.apache.eventmesh.dashboard.console.modle.dto.config;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+public class DynamicGetConfigDTO {
```
[the remark this answers is not in the document]
Comment (zzxxiansheng) [reply eventmesh-dashboard-console/src/main/java/org/apache/eventmesh/dashboard/console/modle/dto/config/DynamicGetConfigDTO.java:27]: They are usually wrapped in a request class, dynamic means that the parameters in it are dynamic lookup parameters

[the remark this answers is not in the document]
Comment (Pil0tXia) [reply eventmesh-dashboard-console/src/main/java/org/apache/eventmesh/dashboard/console/modle/dto/config/DynamicGetConfigDTO.java:27]: Yeah, usually the params in `XXXRequest` are all dynamic, so emphasis 'dynamic' in class name is unnecessary.

[the remark this answers is not in the document]
Comment (zzxxiansheng) [reply eventmesh-dashboard-console/src/main/java/org/apache/eventmesh/dashboard/console/modle/dto/config/DynamicGetConfigDTO.java:27]: oh，l get it.

**Comment by Pil0tXia** [comment]
Please resolve conficts~

File: eventmesh-dashboard-console/src/main/java/org/apache/eventmesh/dashboard/console/enums/metadata/MetadataServiceTypeEnum.java
Comment (Pil0tXia) [inline eventmesh-dashboard-console/src/main/java/org/apache/eventmesh/dashboard/console/enums/metadata/MetadataServiceTypeEnum.java:1]: Please refer to https://github.com/apache/eventmesh-dashboard/pull/85#discussion_r1554814795 and sync changes of https://github.com/apache/eventmesh-dashboard/pull/85.

**Review by Pil0tXia — APPROVED** [review]

**Pull request merged**